.goreleaser.yml

```diff
@@ -15,6 +15,8 @@ builds:
       - windows
       - darwin
     main: ./cmd/node/main.go
+    ldflags:
+    - -s -w -X cqfn.org/degitx/gitlab/service/server.Version={{.Version}}
```

Comment:
@stepanvalyavskiy I don't think it's the correct place (`cqfn.org/degitx/gitlab/service`) to keep DeGitX version. We may use it from another places later, e.g. from CLI by `node --version`, so I'd put it in a root package (`cqfn.org/degitx`) instead of Gitaly specific